src/style-spec/reference/v8.json

```diff
@@ -4450,6 +4450,66 @@
       },
       "property-type": "data-driven"
     },
+    "circle-ang-start": {
+      "type": "number",
+      "default": 0,
+      "minimum": 0,
+      "transition": true,
+      "units": "angle",
+      "doc": "Angle value to start drawing a circle.",
```

Comment:
Could you update the documentation to explain in which quadrant it starts, which direction it is going in and update the unit to degrees instead of angle?

Reply:
and ideally what it's relative to (map grid north instead of viewport)?